docs/tutorials/fundamentals/viewer.md

```diff
@@ -106,28 +106,25 @@ We'll go through each of these in the next sections.
 
 ### Main menu or top bar menu
 
-The main menu consists of the **File**, **View**, **Window**, **Plugins**, and **Help** options.
+The main menu consists of the **File**, **View**, **Layers**, **Plugins**, **Window** and **Help** options.
 
-* **File** has the options to open files, folders, and samples, save layers and screenshots, copy screenshots to clipboard and, in the Windows version, preferences. Additionally, you can make a new `Image` layer from an image (or URL to an image) copied to your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
+* **File** has the options to open files, folders, and samples, save layers and screenshots, copy screenshots to clipboard and, in the Windows and Linux versions, preferences. Additionally, you can make a new `Image` layer from an image (or URL to an image) copied to your Clipboard (keybinding {kbd}`Command/Ctrl+N`).
 
-    All the options on the **File** menu are relatively self-explanatory except **Preferences** on the Windows version of napari. **Preferences** allows you to personalize napari to some degree. To learn more about the **Preferences** menu, there is a tutorial designed for developers [here](https://napari.org/stable/guides/preferences.html).
+    All the options on the **File** menu are relatively self-explanatory except **Preferences** on the Windows and Linux versions of napari. **Preferences** allows you to personalize napari to some degree. To learn more about the **Preferences** menu, see our [Preferences guide for developers](napari-preferences).
 
     **Note:** In macOS, **Preferences** is under the napari menu.
 
-
 * **View** allows you to toggle full screen, the menu bar, play, display axes, the scale bar, tooltips, and the activity dock.
-
-* **Window** allows you to open the integrated console, display the layer controls and layer list.
-
+* **Layers** is a placeholder menu, to be used by plugins to add their own menu items with actions related to layers.
```

Comment:
```suggestion
* **Layers** contains actions and commands designed to act on existing layers, or generate new ones. The menu is mostly designed for use by plugins, but will also be populated with builtin actions. Currently this menu is **in development**, so items may change from version to version. For more information on this menu and its usage, see NAP-6
```

Reply:
@melissawm suggested more detail here. I wasn't sure how to do the ref to NAP-6. Ping when this is otherwise ready for review!